core/vtk/ttkImportEmbeddingFromTable/ttkImportEmbeddingFromTable.h

```diff
@@ -83,20 +41,11 @@ class TTKIMPORTEMBEDDINGFROMTABLE_EXPORT ttkImportEmbeddingFromTable
   vtkSetMacro(Embedding2D, bool);
   vtkGetMacro(Embedding2D, bool);
 
-  int FillInputPortInformation(int port, vtkInformation *info) override {
-    if(port == 0)
-      info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkPointSet");
-    if(port == 1)
-      info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkTable");
-
-    return 1;
-  }
-
 protected:
   ttkImportEmbeddingFromTable() {
-    UseAllCores = true;
-
+    this->setDebugMsgPrefix("ttkImportEmbeddingFromTable");
```

Comment:
note: the other modules omit the ```ttk``` prefix in their name. I'll fix that later.

Reply:
I thought the other modules omit the ttk prefix in the base layer and it is added automatically in the vtk layer. Is that wrong?